setup.cfg

```diff
@@ -34,25 +34,7 @@ omit =
  tests/*
 
 [tool:pytest]
-addopts = --flake8
-flake8-max-line-length = 120
```

Comment:
is there another place where we set the line length?

Reply:
Yes, further down in the flake8 section. See the full file: https://github.com/Open-MSS/MSS/blob/5a8d1c83ef2b763696fe94f93a424f52bfbb2d05/setup.cfg